modules/gdscript/gdscript_tokenizer.cpp

```diff
@@ -56,9 +56,12 @@ static const char *token_names[] = {
 	// Logical
 	"and", // AND,
 	"or", // OR,
+	"but", // BUT,
 	"not", // NOT,
 	"&&", // AMPERSAND_AMPERSAND,
 	"||", // PIPE_PIPE,
+	"^^", // CARET_CARET,
+	"xor", // XOR,
```

Comment:
The order of the names in the array must be the same as in the enum (`gdscript_tokenizer.h`).